lib/models/asset-size-printer.js

```diff
@@ -49,12 +49,12 @@ module.exports = class AssetPrinterSize {
 
   makeAssetSizesObject() {
     return new Promise((resolve) => {
```

Comment:
I think we can remove this manual `new Promise` wrapper now

Reply:
done

Reply:
Hmm, I don't understand what you mean. How is this done?

Reply:
My point above was that we should make the `makeAssetSizesObject` method itself an async function, then we don't need to `return new Promise` at all (but we can use `async`/`await` to wait for the `pool.exec` results.